src/Orleans.Transactions/DistributedTM/TransactionInfo.cs

```diff
@@ -39,8 +39,6 @@ public TransactionInfo(TransactionInfo other)
             this.IsReadOnly = other.IsReadOnly;
             this.TimeStamp = other.TimeStamp;
             this.Priority = other.Priority;
-            this.TMCandidate = other.TMCandidate;
-            this.TMBatchSize = other.TMBatchSize;
```

Comment:
I understand TMCandidate has been moved to TransactionInfo? But where are we setting TMBatchSize now? 
I guess my question is , why is this removed?

Reply:
Batch size was an optimization for TM selection.  It was not strictly necessary, but may influence performance.  I suspect it's a premature optimization, but we'll see.  If we get perf drop from this, we can revisit.

Reply:
yep, we don't have super strong evidence that this optimization is important in general. I have seen it make a big difference in some cases but in the long run we may be better off just giving the application some control over TM selection than trying to do it automatically.